docs/integrations/builtin/credentials/coda.md

```diff
@@ -12,7 +12,7 @@ You can use these credentials to authenticate the following nodes with Coda.
 
 ## Prerequisites
 
-Create a [Coda](https://www.coda.com/) account.
+Create a [Coda](https://www.coda.io/) account.
```

Comment:
```suggestion
Create a [Coda](https://www.coda.io/){:target=_blank .external-link} account.
```

Reply:
Open external links in new tab. I realise it didn't have this before - this is something we're fixing as we see instances of it.

Reply:
Oh yeah you are totally right! I didn't think of adding it, but will now remember for sure 😉